### PR TITLE
Fix for elastic skin

### PR DESCRIPTION
**Pull request by ScottPeterJohnson**
Tentative fix for #26, adds support for the Elastic skin.

### CODE REVIEW

**Comment by photoninger** [comment]
This patch works only for english users, I changed the regex to catch at least all one-word translations of "from":
`return array('content' => preg_replace('/(<span>\s*)()(\w+)\s+<span class="adr">/', '$1' . $this->img_status . ' $3 ', $p['content']));`

File: authres_status.php [hunk 1]
```diff
@@ -240,13 +240,22 @@ public function message_headers($p)
...
+        $this->populate_message_headers($p);
 
         $p['output']['from']['value'] = $this->img_status . $p['output']['from']['value'];
         $p['output']['from']['html'] = true;
 
         return $p;
     }
 
+    public function message_summary($p){
+        $this->populate_message_headers($GLOBALS['MESSAGE']);
```
Comment (pimlie) [inline authres_status.php]: Didnt look this up in the docs but is using $GLOBALS really required?

File: authres_status.php [hunk 2]
```diff
@@ -240,13 +240,22 @@ public function message_headers($p)
...
+        return array('content' => preg_replace('/(<span>\s*)()(From)/', '$1' . $this->img_status . ' $3', $p['content']));
```
Comment (pimlie) [inline authres_status.php]: Please have a look at the comment/regex provided by @photoninger

**Comment by pimlie** [comment]
Closing this PR as support has been added in #37, thanks for your contrib though!

**Pull request closed without merging**